### ISSUE TITLE
Need help. Errors due to nulls.

### ISSUE THREAD

**Issue by mikend** [vuejs/Discussion#134]
Errors have suddenly started to appear in the app I'm developing. It has been working for months, and I can't really trace them to any changes I've made lately. It's a pretty large app and the troubles happen when I change views and I can't really wrap my head around it.

Anyhow, I guess the errors are somehow related, and hopefully someone can point me in the right direction.

So this is some of the stuff that fails:

Line 1548, exports.$addChild: this._children is null 
Line 4272, Directive.protoype._teardown: this.vm._watchers is null
Line 6436, exports.compile: vm._children is null

Thanks!

**Comment by yyx990803** [comment]
Looks like methods are being called on an already destroyed vm instance. Did this happen after a Vue upgrade? What version are you using?

**Comment by mikend** [comment]
At first I was using 0.11.3, then I upgraded to .5 and the problems remained. Currently I'm using a version of .5 that I hacked a bit to ignore the errors and everything seems to work fine.

What I did when the problems first appeared was that I implemented a permissions system, basically adding a bunch of v-if attributes checking a permissions object that's inherited from the root component. So basically I just added a few more levels of conditional Vue components, all of them inherit:all.

I will try to isolate the problem (if there is one) and post it somewhere.
